addons/analytic/static/src/components/analytic_distribution/analytic_distribution.js

```diff
@@ -87,7 +87,7 @@ export class AnalyticDistribution extends Component {
         // and thus different applicabilities apply
         // or a model applies that contains unavailable plans
         // This should only execute when these fields have changed, therefore we use the `_field` props.
-        const valueChanged = JSON.stringify(this.props.value) !== JSON.stringify(nextProps.value);
+        const valueChanged = JSON.stringify(this.props.record.data[this.props.name]) !== JSON.stringify(nextProps.record.data[nextProps.name]);
```

Comment:
Not sure this will work, maybe we need to create this.currentValue that is started, and changed at the end of the willUpdate